clips/dry/README.md

```diff
@@ -0,0 +1,21 @@
+# Rachid Zarouali / clips / dry
+
+Run [dry](https://github.com/moncho/dry) and explore contents of the image layers.
+
+![](https://raw.githubusercontent.com/lukaszlach/clip/master/clips/dive/record.gif)
+
+```
+docker dry -e DOCKER_HOST=$DOCKER_HOST
```

Comment:
Remove the `-e` as plugin passes no parameters to the dry tool, so accepts none itself